I18n fr (#460)
InnerSourceCommons/innersourcecommons.org: content/fr/community/_index.md

---
title: "Communauté"
---

<section class="section">
  <div class="container">
    <div class="row align-items-center">
      <div class="col-md-6 order-2 order-md-1">

>         <p>Bienvenue sur la page de la Communauté - nous sommes heureux que vous soyez ici ! Toutes nos ressources d'apprentissage et tous nos événements sont ouverts à tous et leur utilisation est gratuite. Toutefois, la magie opère lorsque la communauté se réunit pour partager ses connaissances. Nous espérons que vous vous joindrez à nous !

Reviewer: suggestion: nous sommes heureux de vous voir ici

Author: Taken into account in next commit

        </div>
      </div>
      <div class="col-md-6 ">
        <p class="section-title h2">Groupe de travail sur les parcours d'apprentissage (Les "Learning Paths")</p>

>         <p>Le Groupe de travail sur <a href="/learn/learning-path/">les Learning Paths </a>, produit une série de courtes vidéos et d'articles expliquant et enseignant divers aspects de l'InnerSource. Le groupe de travail se réunit chaque semaine. Pour commencer, rejoignez le canal slack #learning-path et présentez-vous.

Reviewer 2: ```suggestion
        <p>Ce Groupe de travail regroupe sur les <a href="/learn/learning-path/">InnerSource Learning Paths </a>  une série de courtes vidéos et d'articles expliquant et enseignant divers aspects de l'InnerSource. Le groupe de travail se réunit chaque semaine. Pour commencer, rejoignez le canal slack #learning-path et présentez-vous.
```

Author: I don't agree, the meaning of your version is not the same as initial version

  <div class="container">
    <div class="row align-items-center">
      <div class="col-md-6 order-2 order-md-1">

>         <p class="section-title h2">Conseil & Gouvernance</p>

Reviewer 2: ```suggestion
        <p class="section-title h2">Conseil d'administration & Gouvernance</p>
```

Author: We call it "Conseil d'administration" everywhere else, I prefer to keep consistency

Author: sorry, you were right, fixed

>         <p>InnerSource Commons est un organisme à but non lucratif (501(c), et le service au Conseil d'administration est un moyen pour la communauté de renforcer son engagement envers l'InnerSource.

Reviewer 2: ```suggestion
        <p>InnerSource Commons est une fondation américain à but non lucratif (501(c)(3) selon le code des impôts US), et le service au Conseil d'administration est un moyen pour la communauté de renforcer son engagement envers l'InnerSource.
```

Author: fixed

Author: la meilleure traduction pour moi est bien "organisme à but non lucratif" en Français si on regarde l'équivalent du 501 US

  <div class="container">
    <div class="row justify-content-center">
      <div class="col-12 text-center">
        <p class="section-title mb-5 mt-3 h1">Comment nous travaillons</p>

>         <p>L'affiliation à l'ISC est liée aux individus, et non aux entreprises. Vous trouverez ci-dessous quelques détails sur notre mode de fonctionnement. Aucun frais n'est exigé pour participer. Le contenu est mis à disposition sous la Licence <a href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.</p>

Reviewer 2: ```suggestion
        <p>Seuls des individus peuvent être affiliés à l'ISC, pas des entreprises. Vous trouverez ci-dessous quelques détails sur notre mode de fonctionnement. La participation est gratuite. Le contenu est mis à disposition sous la licence <a href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.</p>
```

Author: Sure, your version is better, I keep it

      </div>
      <div class="col-md-4 col-sm-6 mb-4">
        <div class="feature-card text-center bg-light">
          <i class="ti-bookmark-alt mb-3"></i>
          <h4 class="mb-2">Outils & Ressources</h4>
          <p>La plupart de nos discussions se déroulent sur Slack. Nous avons des projets publics et privés sur <a href="http://github.com/InnerSourceCommons">GitHub</a> et nous stockons beaucoup de nos documents de travail dans gDrive.
          </p>
        </div>
      </div>
      <div class="col-md-4 col-sm-6 mb-4">
        <div class="feature-card text-center bg-light">
          <i class="ti-comments mb-3"></i>
          <h4 class="mb-2">Chatham House Rule</h4>

>           <p>Toutes les discussions dans notre canal Slack se déroulent sous la règle dite du <a href="https://www.chathamhouse.org/about-us/chatham-house-rule">Chatham House Rule</a> pour créer un espace sûr où les gens peuvent poser toutes leurs questions.</p>

Reviewer 2: ```suggestion
          <p>Toutes les discussions dans notre canal Slack sont soumises à la règle dite du <a href="https://www.chathamhouse.org/about-us/chatham-house-rule">Chatham House Rule</a> pour créer un espace sûr où les gens peuvent poser toutes leurs questions.</p>
```

Author: fixed

        </div>
      </div>
      <div class="col-md-4 col-sm-6 mb-4">
        <div class="feature-card text-center bg-light">
          <i class="ti-face-smile mb-3"></i>
          <h4 class="mb-2">Code de Conduite</h4>

>           <p>Soyez gentil et poli. Si vous souhaitez obtenir plus de détails, veuillez lire notre <a href="/about/codeofconduct/">Code de Conduite</a>.</p>

Reviewer: Suggestion : Soyez bienveillant et courtois

Author: Sure :) this is much better, taken into account in next commit.

        <a class="twitter-timeline" data-height="500" data-dnt="true" href="https://twitter.com/InnerSourceOrg?ref_src=twsrc%5Etfw"></a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
      </div>
      <div class="col-md-5 offset-md-1">

>         <p class="h2 section-title">Rester connecté</p>

Reviewer 2: ```suggestion
        <p class="h2 section-title">Gardons le contact</p>
```

Author: fixed

>         <p class="mb-4">La plupart de la vie de la communauté se déroule sur notre <a href="https://join.slack.com/t/innersourcecommons/shared_invite/zt-1l4a602b6-DKD0B0Y8~WE4aDf~o1xCRw">Canal Slack</a>. Vous pouvez également vous tenir au courant des nouvelles, des événements et des activités d'InnerSource Commons en vous inscrivant à notre liste de diffusion ci-dessous, en nous suivant sur <a href="https://twitter.com/InnerSourceOrg">Twitter</a> et <a href="https://www.linkedin.com/company/innersourcecommons">LinkedIn</a>, at en suivant notre <a href="https://www.youtube.com/channel/UCoSPSd6Or4F_vpjo4SmyoEA">Chaine YouTube </a>.</p>

Reviewer 2: ```suggestion
        <p class="mb-4">La plupart de la vie de la communauté se déroule sur notre <a href="https://join.slack.com/t/innersourcecommons/shared_invite/zt-1l4a602b6-DKD0B0Y8~WE4aDf~o1xCRw">canal Slack</a>. Vous pouvez également vous tenir au courant des nouvelles, des événements et des activités d'InnerSource Commons en vous inscrivant à notre liste de diffusion ci-dessous, en nous suivant sur <a href="https://twitter.com/InnerSourceOrg">Twitter</a> et <a href="https://www.linkedin.com/company/innersourcecommons">LinkedIn</a>, et en suivant notre <a href="https://www.youtube.com/channel/UCoSPSd6Or4F_vpjo4SmyoEA">chaîne YouTube </a>.</p>
```

Author: Same answer as before, keep the uppercase in that particular circonstance
